easybuild/easyblocks/a/anaconda.py

```diff
@@ -42,33 +42,11 @@
 
 def set_conda_env(installdir):
     """ Set the correct environmental variables for conda """
-    myEnv = os.environ.copy()
-    env.setvar('PATH', "{}/bin".format(installdir) + ":" + myEnv["PATH"])
+    env.setvar('PATH', os.path.join(installdir, 'bin') + ':' + os.environ.get('PATH', ''))
```

Comment:
@jerowe I'm not sure how this worked for you, without this change, I run into this:

```
  File "/tmp/eb-xePLqv/included-easyblocks/easybuild/easyblocks/anaconda.py", line 46, in set_conda_env
    env.setvar('PATH', "{}/bin".format(installdir) + ":" + myEnv["PATH"])
ValueError: zero length field name in format
```

replacing `{}` with `{0}` would probably have worked too